src/python/pants/core/util_rules/environments.py

```diff
@@ -54,6 +54,8 @@ class EnvironmentsSubsystem(Subsystem):
 # Environment targets
 # -------------------------------------------------------------------------------------------
 
+LOCAL_ENVIRONMENT_MATCHER = "__local__"
```

Comment:
@cognifloyd recommended in Slack that we use `__local__` over `<local>`. I think it makes sense. It's consistent with `__defaults__` in BUILD files, and `__default__` for the new `[python].resolves_to_x` options.